packages/repository/src/types/strategyProcessingCheckpoint.types.ts

```diff
@@ -0,0 +1,15 @@
+import { ChainId, Hex } from "@grants-stack-indexer/shared";
+
+export type StrategyProcessingCheckpoint = {
+    chainId: ChainId;
+    strategyId: Hex;
+    lastProcessedBlockNumber: number;
+    lastProcessedLogIndex: number;
+    createdAt?: Date;
```

Comment:
left it optional because I delegated (in this case) the default to `sql.now()` instead of doing it in the repository. Aldo at business level is not so critical but is nice to have it there on the DB